src/ucp/core/ucp_context.c

```diff
@@ -639,6 +642,19 @@ static ucs_status_t ucp_fill_resources(ucp_context_h context,
          * client-server connection establishment via sockaddr, don't use it */
         if ((num_tl_resources > 0) ||
             (context->tl_mds[md_index].attr.cap.flags & UCT_MD_FLAG_SOCKADDR)) {
+            /*List of memory type MDs*/
```

Comment:
please pick a single memory domain for each memtype. will both gdrcopy and codacopy be placed in this array?

Reply:
currently yes.   but it is optimal to avoid duplicates in multiple active memoty types  case.
will fix this.